### ISSUE TITLE
F# example does not compile because "Path not exists: obj\Debug\Grains.dll"

### ISSUE THREAD

**Issue by davidpodhola** [dotnet/orleans#154]
First please understand I just started with Orleans.

I installed 1.0.1 (the latest version available), open/no edit/close to fix Env. path Visual Studio Issue, downloaded FSharpHelloWorld from https://orleans.codeplex.com/SourceControl/latest

When I try to build the solution, project Grains does not compile. Visual Studio Output window reveals that step

```
[OrleansServerObjProcessing] - Creating Server for Project=Grains
```
warns about
    
```
MISSING: Path not exists: obj\Debug\Grains.dll
```
later on

```
    Initializing serializer generation manager
    ERROR -- Client code-gen FAILED -- Exception caught -- 
    Exception = System.IO.FileNotFoundException
    Grains.dll ---> System.IO.FileNotFoundException
```

File 
```
obj\Debug\Grains.dll
```
really does not exist, I seems like `ClientGenerator.exe` is run before the library is compiled. Please advise, thanks!

**Comment by sergeybykov** [comment]
Did you clone the sample code or download it from Codeplex as a zip file? In the latter case, did you unblock the zip (right-click Unblock) after downloading and before extracting? I just did that, and after fixing up the API names that changed since the Preview, I got the sample up and running.

Now that fixed it, I should add it to GitHub.

**Comment by sergeybykov** [comment]
I just submitted a PR to migrate the sample - https://github.com/dotnet/orleans/pull/155.

**Comment by gabikliot** [comment]
Closed via #155.

**Comment by davidpodhola** [comment]
Thanks. It is much better, but I still encounter issues. Yes, I downloaded the sample code from Codeplex as a zip file, but there was no need to unblock it.

The issue is probably related to the fact I used path with spaces (" "). Fsc.exe was run here with

``` 
-o:C:\Users\david.podhola\Documents\Visual Studio 2013\Projects\orleans\Samples\FSharpHelloWorld\Grains\obj\Debug\Grains.dll
```
When I included the quotation marks, it compiles correctly. I moved the solution to C:\temp and again, it compiles correctly.

I will submit PR.
